include/tlog/source.h

```diff
@@ -32,7 +32,10 @@
 
 /** Abstract source */
 struct tlog_source {
-    const struct tlog_source_type *type;   /**< Type */
+    const struct tlog_source_type *type;           /**< Type */
+#ifndef NDEBUG
+    struct timespec                last_timestamp; /**< Timestamp */
```

Comment:
Since we have comments here, it would be better to make them at least match the specificity of the variable name. I.e. could you please write "Last timestamp" here instead of just "Timestamp"?